features/visitor_is_able_to_sign_up.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Visitor is able to sign up
+    As a User
+    In order to access the mailboxer
+    I want to be able to Sign Up
+
+Scenario: User can sign up
```

Comment:
Scenarios should be indented to so it lines up with the user story

Reply:
done